install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -24,8 +24,17 @@ type Config struct {
 }
 
 type CommonConfig struct {
-	PodConfig                map[string]*PodConfig `json:"podConfig,omitempty"`
-	StaticMessagebusPassword string                `json:"staticMessagebusPassword"`
+	PodConfig map[string]*PodConfig `json:"podConfig,omitempty"`
+	// ServiceConfig is only supported for "proxy" service at the moment
+	ServiceConfig            map[string]*ServiceConfig `json:"serviceConfig,omitempty"`
+	StaticMessagebusPassword string                    `json:"staticMessagebusPassword"`
+}
+
+// ServiceConfig enables modification of type of service to `ClusterIP` if needed,
+// currently supported only for `proxy` service
+type ServiceConfig struct {
```

Comment:
If this is special to proxy: Why not add it to `ProxyConfig` ([link](https://github.com/gitpod-io/gitpod/pull/10537/files#diff-7480b9126452b70b6c97afd8fbdcf95a8b614a986c02c113aafc5f691fef5630R165)), then?

Mentioning this because we have tried to keep things simple and non-generic in the installer to start with. And only extend to general solutions where it solved multiple use-cases. But I don't think we have another service that needs the same type of variance in the near future.

Reply:
Also happy to sync to unblock here! :slightly_smiling_face:

Reply:
Yikes, I acted on a whim :see_no_evil: Have updated the setup now. Sorry for the back-and-fro.

Reply:
Awesome, thx for addressing! :pray: